lib/curl_ntlm_core.h

```diff
@@ -53,6 +53,10 @@
 #define USE_NTLM2SESSION 1
 #endif
 
+#if !defined(USE_NTLM2SESSION)
+#define USE_NTLM2SESSION 0
```

Comment:
Depending on how libcurl is built with ntlm `USE_NTLM2SESSION` is defined 0 or 1 or undefined. In ntlm code I notice there is a lot of #if FOO instead of #if defined(FOO). For example there is [this](https://github.com/curl/curl/blob/curl-7_53_1/lib/vauth/ntlm.c#L397) twice:
`#if USE_NTRESPONSES && USE_NTLM2SESSION`
He has a compiler switch that catches when FOO macro is evaluated to 0 due to undefined FOO